Microsoft.Toolkit.Parsers/Markdown/MarkdownDocument.cs

```diff
@@ -55,7 +55,7 @@ public MarkdownDocument()
         /// <param name="markdownText"> The markdown text. </param>
         public void Parse(string markdownText)
         {
-            Blocks = Parse(markdownText, 0, markdownText.Length, quoteDepth: 0, actualEnd: out int actualEnd);
+            Blocks = Parse(markdownText, 0, markdownText.Length, quoteDepth: 0, actualEnd: out _);
```

Comment:
What about an override that doesn't include the out variable?

Reply:
It is doable but may not bring a lot of value. I've seen if we can remove the `out` parameter but it is still used and a refactoring is out of scope.